bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/ArrayValueImpl.java

```diff
@@ -228,7 +229,7 @@ public ArrayValueImpl(ArrayType type, long size) {
         }
     }
 
-    public ArrayValueImpl(ArrayType type, long size, ListInitialValueEntry[] initialValues) {
+    public ArrayValueImpl(ArrayType type, long size, BInitialValueEntry[] initialValues) {
```

Comment:
Why do we change the initial values into the abstract type here? 
Because `BInitialValueEntry` is common for both list and mapping values. 
Wouldn't it be more specific to keep them as `ListInitialValueEntry[]` for arrays & tuples?